src/main/java/peoplesoft/logic/commands/job/JobAssignCommand.java

```diff
@@ -36,6 +37,7 @@
 
     public static final String MESSAGE_SUCCESS = "Assigned Job %s to %s\n%s has the following jobs: %s";
     public static final String MESSAGE_JOB_NOT_FOUND = "This job does not exist";
+    public static final String MESSAGE_DUPLICATE_EMPLOYMENT = "This person is already assigned to this job";
 
     private ID jobId;
     private Index personIndex;
```

Comment:
should we use personId here?

Reply:
I was thinking of using `Index` for both, based off what you see on the job/person lists.

Reply:
ideally we should be decoupling list index from person IDs, cuz indexes can change very frequently (and iirc the UI displays person IDs?)

Reply:
the `{has,get}{Person,Job}` methods in `Model` should make it easier, i think

Reply:
Ok but I'm thinking of implementing this in a different branch so ill do it there.